tests/test_workflow.py

```diff
@@ -41,10 +71,48 @@ def test_full_workflow():
         assert exists(f)
     # check that the files are big enough
     for f in expected_files:
-        assert os.stat(f).st_size > 400
+        assert os.stat(f).st_size > 300
     # then do the presence/absence estimation
     if exists(abundance_file):
         os.remove(abundance_file)
+    # python ../run_YACHT.py --json testdata/20_genomes_trained_config.json --sample_file testdata/sample.sig.zip --out_file result.xlsx --outdir testdata/
+    cmd = f"python {os.path.join(script_dir, 'run_YACHT.py')} --json {os.path.join(data_dir, '20_genomes_trained_config.json')} --sample_file {sample_sketches} --significance 0.99 --min_coverage 0.001 --outdir {data_dir} --out_filename {abundance_file} --show_all"
+    print(cmd)
```

Comment:
Same comment re: removal of unnecessary code comments

Reply:
I have deleted them.